.github/workflows/package.yml

```diff
@@ -126,23 +126,41 @@ jobs:
       with:
         node-version: 20
         registry-url: ${{ matrix.registry }}
+        always-auth: true
+        token: ${{ secrets[matrix.token-name] }}
+    
+    - name: Manually Configure npm Authentication
+      run: |
+        REGISTRY="${{ matrix.registry }}"
+        CLEAN_REGISTRY=${REGISTRY#https://}  # Strip https:// prefix
+        echo "//${CLEAN_REGISTRY}/:_authToken=${{ secrets[matrix.token-name] }}" > ~/.npmrc
+        echo "Generated .npmrc:"
+        cat ~/.npmrc  # Debug output
+        NODE_AUTH_TOKEN=${{ secrets[matrix.token-name] }}
```

Comment:
Whats brought you to hear @robmoffat? In the old version the setup step on line 128 is setting the registry URL and line 150 the token (via an environment variable), which I know was working... We also did see beta.1 published to NPM successfully - I believe it was just the build that was broken in that case, right?

Reply:
Yes, publishing was working fine, it was just empty package because of includes or directory structure issues.

Reply:
Separate problems I think.  What I'm doing above was trying to fix the npm install.  It fails before it even gets to publishing.  This didn't work either.